Client/src/Components/Sidebar/index.jsx

```diff
@@ -498,14 +502,33 @@ function Sidebar() {
               className={
                 location.pathname.includes(item.path) ? "selected-path" : ""
               }
-              onClick={() =>
-                item.path === "support"
-                  ? window.open(
+              onClick={
+                () => {
+                  if (item.path === "support") {
+                    window.open(
                       "https://github.com/bluewave-labs/bluewave-uptime/issues",
                       "_blank",
                       "noreferrer"
-                    )
-                  : navigate(`/${item.path}`)
+                    );
+                  }
+                  else if(item.path === "docs"){
+                    window.open(
+                            "https://bluewavelabs.gitbook.io/uptime-manager",
+                            "_blank",
+                            "noreferrer"
+                          )
+                  }
+                  else if(item.path === "changelog"){
+                    window.open(
+                            "https://github.com/bluewave-labs/bluewave-uptime/releases",
+                            "_blank",
+                            "noreferrer"
+                          )
+                  }
+                  else{
+                    navigate(`/${item.path}`)
```

Comment:
While this works, I'm not a big fan of `if/else` ladders, especially if it involves repeated logic.  Here we have the `window.open` logic repeated three times, not ideal from a maintainability perspective.

How about the following?  Declare a module scoped URL_MAP that we can use:

```
const URL_MAP = {
  support: "https://github.com/bluewave-labs/bluewave-uptime/issues",
  docs: "https://bluewavelabs.gitbook.io/uptime-manager",
  changelog: "https://github.com/bluewave-labs/bluewave-uptime/releases",
};
```

Now we can get refactor the `onClick` logic:

```
onClick={() => {
  const url = URL_MAP[item.path];
  if (url) {
    window.open(url, "_blank", "noreferrer");
  } else {
    navigate(`/${item.path}`);
  }
}}
```

That's cleaner and more maintanable if we ever decide to change the behavior when one of the links is selected.